packages/flutter/lib/src/rendering/table.dart

```diff
@@ -897,7 +896,7 @@ class RenderTable extends RenderBox {
     else if (tableWidth < minWidthConstraint) {
       final double delta = (minWidthConstraint - tableWidth) / columns;
       for (int x = 0; x < columns; x += 1)
-        widths[x] += delta;
+        widths[x] = widths[x]! + delta;
       tableWidth = minWidthConstraint;
     }
 
```

Comment:
if you remove the assert just below here, you can make unFlexedTableWidth non-nullable